mesheryctl/pkg/utils/error.go

```diff
@@ -306,6 +304,52 @@ func ErrInvalidNameOrID(err error) error {
 		[]string{"Provided Name or ID would be invalid"}, []string{"Please Provide a valid name or ID by using list command with the appropriate subcommand"})
 }
 
+func ErrAttachAuthToken(err error) error {
+	return errors.New(ErrAttachAuthTokenCode, errors.Alert, []string{"Authentication token not found"},
+		[]string{"Authentication token not found", err.Error()}, []string{"User not logged in to generate token"}, []string{"Login with `mesheryctl system login` or supply a valid user token using the --token (or -t) flag."})
+}
+
+func ErrFailReqStatus(statusCode int) error {
+	return errors.New(ErrFailReqStatusCode, errors.Alert, []string{"Failed response Server Error"},
+		[]string{"Response Status Code " + strconv.Itoa(statusCode) + ", possible Server Error"}, []string{"Bad api call"}, []string{"Check your network connection and check the status of meshery server via 'mesheryctl system status'"})
```

Comment:
1. Don't capitalize Server Error.
2. Don't use "bad" as an adjective here. Use "Invalid" instead.
3. Do capitalize API.
4. Always capitalize Meshery and its components' names: Meshery Server.
5. Don't use single quotes, when you should be using backticks: `mesheryctl system status`.